src/Entities/Issue.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace ITBugTracking\Entities;
+
+use JsonSerializable;
+
+class Issue implements JsonSerializable
+{
+    public int $id;
+    public string $title;
+    public string|null $description;
+    public string $severity;
+    public string $date_created;
+    public string $reporter;
+    public int $department;
+    public int $completed;
+    public int|null $issue_id;
+    public int|null $comment_count;
+
+    public function jsonSerialize(): mixed
+    {
+        return [
+            'id' => $this->id,
+            'title' => $this->title,
+            'summary' => substr($this->description, 0, 100),
```

Comment:
Could this be done more efficiently in the SQL query?

Reply:
Done